src/register/RegistrationPage.jsx

```diff
@@ -72,24 +73,31 @@ class RegistrationPage extends React.Component {
       failureCount: 0,
       startTime: Date.now(),
       totalRegistrationTime: 0,
-      optimizelyExperimentName: '', // eslint-disable-line react/no-unused-state
+      optimizelyExperimentName: '',
       readOnly: true,
       validatePassword: false,
-      // TODO: Remove after VAN-704 is complete
+      // TODO: Remove after VAN-876 experimentation is complete.
       registerRenameExpVariation: '',
     };
   }
 
   componentDidMount() {
     window.optimizely = window.optimizely || [];
-    window.optimizely.push({
-      type: 'page',
-      pageName: 'authn_registration_page',
-      isActive: true,
-    });
-
```

Comment:
Lets keep this as it is. We don't need to move it behind a condition. If we don't need it for this experiment we can skip it in Optimizely metric.